cirq/aqt/aqt_device.py

```diff
@@ -26,50 +26,32 @@
 import json
 from typing import Union, Tuple, List, Dict, Sequence, Any, cast
 import numpy as np
-from cirq import ops, devices, study
-from cirq import Circuit, LineQubit, IonDevice, Duration
-from cirq import DensityMatrixSimulator
 
-gate_dict = {'X': ops.X, 'Y': ops.Y, 'Z': ops.Z, 'MS': ops.XX}
+from cirq.ops import MeasurementGate, \
+    X, Y, Z, XX,\
+    ZPowGate, XPowGate, YPowGate, XXPowGate,\
+    Moment, Operation, Qid,\
+    depolarize, measure
 
+from cirq import Circuit, Duration, DensityMatrixSimulator
 
-def get_op_string(op_obj: ops.Operation) -> str:
-    """Find the string representation for a given gate
+from cirq.devices import LineQubit, NoiseModel, NO_NOISE
 
-    Args:
-        op_obj: Gate object, one of: XXPowGate, XPowGate, YPowGate, ZPowGate
+from cirq.ion import IonDevice
 
-    Returns:
-        String representing the gate operations
-    """
-    if isinstance(op_obj, ops.XXPowGate) or isinstance(op_obj.gate,
-                                                       ops.XXPowGate):
-        op_str = 'MS'
-    elif isinstance(op_obj, ops.XPowGate) or isinstance(op_obj.gate,
-                                                        ops.XPowGate):
-        op_str = 'X'
-    elif isinstance(op_obj, ops.YPowGate) or isinstance(op_obj.gate,
-                                                        ops.YPowGate):
-        op_str = 'Y'
-    elif isinstance(op_obj, ops.ZPowGate) or isinstance(op_obj.gate,
-                                                        ops.ZPowGate):
-        op_str = 'Z'
-    elif isinstance(op_obj, ops.MeasurementGate) or isinstance(
-            op_obj.gate, ops.MeasurementGate):
-        op_str = 'Meas'
-    else:
-        raise ValueError('Got unknown gate:', op_obj)
-    return op_str
+from cirq.study import TrialResult
+
+gate_dict = {'X': X, 'Y': Y, 'Z': Z, 'MS': XX}
```

Comment:
Global variables should be all capitals.  Also, I would put a comment explaining why this is needed.